dozer-ingestion/src/errors.rs

```diff
@@ -374,12 +377,17 @@ pub enum DebeziumSchemaError {
     #[error("Invalid date")]
     InvalidDateError,
 
+    #[error("Invalid json: {0}")]
+    InvalidJsonError(String),
+
     // #[error("Invalid time")]
     // InvalidTimeError,
     #[error("Invalid timestamp")]
     InvalidTimestampError,
 }
 
+impl Eq for DebeziumSchemaError {}
```

Comment:
cannot be because of 
```
the trait bound `dozer_types::rust_decimal::Error: std::cmp::Eq` is not satisfied [E0277] Note: required by a bound in `std::cmp::AssertParamIsEq`
```

Reply:
If we can't derive, we shouldn't `impl`. Why is it needed?

Reply:
Cause this has been implemented - https://github.com/getdozer/dozer/pull/1200#discussion_r1179868789

Reply:
fixed!